libs/shared/src/types/notification-templates/index.ts

```diff
@@ -0,0 +1,3 @@
+import { CustomDataType } from '../shared';
+
+export type NotificationTemplateCustomData = CustomDataType;
```

Comment:
```suggestion
export type WorkflowCustomData = CustomDataType;
```
To help with our renaming

Reply:
i had it in a lot of places but was not sure what is our policy regarding new items.

Reply:
If it's new, I feel we should just create with the new naming. Wdyt?

Reply:
thanks for the comment i thought it was autogenerated, i will look into it.